### PR TITLE
feat(Filmstrip) add logic resize filmstrip

### PR DESCRIPTION
**Pull request by dudumanbogdan**
- add logic resize filmstrip
- add configuration for resize filmstrip
- remove node-sass and added sass (in order for build to be successful when using node >=14)
- include package-lock

### CODE REVIEW

File: spot-client/package.json [hunk 1]
```diff
@@ -73,6 +72,7 @@
         "react-router-dom": "5.1.2",
         "redux": "4.0.5",
         "redux-thunk": "2.3.0",
+        "sass": "1.54.9",
```
Comment (saghul) [inline spot-client/package.json:75]: Please split this change to a separate commit. One logical change one commit / PR.

File: spot-client/src/common/app-state/config/default-config.js [hunk 2]
```diff
@@ -131,6 +131,26 @@ export default {
...
+     * Configuration object for film strip container size.
+     */
+    FILMSTRIP_SIZE: {
+        /**
+         * The threshold width from where to start changing the filmstrip size.
+         */
+        THRESHOLD_WIDTH: 920,
+
+        /**
+         * The ratio between client width and client height 1:2.
```
Comment (saghul) [inline spot-client/src/common/app-state/config/default-config.js:144]: You mean 2:1, right?

File: spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js [hunk 3]
```diff
@@ -16,6 +16,9 @@ import { WiredScreenshareChangeListener } from '../../wired-screenshare';
 import AbstractMeetingFrame from '../AbstractMeetingFrame';
 import MeetingType from '../MeetingType';
 import FeedbackHider from './FeedbackHider';
+import { getClientAspectRatio,
```
Comment (saghul) [inline spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js:19]: Sort this import correctly, deeper imports go above, after builtins. Didn't the linter catch it?

Also, when importing multiple items in multiple lines, keep each item in a new line.

File: spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js [hunk 4]
```diff
@@ -342,6 +345,9 @@ export class JitsiMeetingFrame extends AbstractMeetingFrame {
             nativeCommands.sendAdjustVolume(data.direction);
             break;
 
+        case COMMANDS.RESIZE_FILMSTRIP:
+            this._jitsiApi.executeCommand('resizeFilmStrip', data);
+            break;
```
Comment (saghul) [inline spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js:350]: style: leave a newline here.

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL.

File: spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js (hunk 3 above)
Comment (dudumanbogdan) [reply spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js:19]: I think I have a problem with the eslint ;( "Error: Failed to load plugin 'flowtype' declared in 'spot-client/.eslintrc » eslint-config-jitsi': Cannot find module 'eslint-plugin-flowtype'"

**Review by saghul — APPROVED** [review]
